backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/SAPGLJournal.java

```diff
@@ -182,6 +183,19 @@ else if (line.isBaseTaxLine())
 				final SAPGLJournalLine taxLine = createTaxLine(line, taxProvider, currencyConverter);
 				it.add(taxLine);
 				hasChanges = true;
+
+				//
+				// once tax line is generated,
+				// calculate line's base tax amount & reset IsTaxIncluded to N
+				if(line.isTaxIncluded())
+				{
+					final Money taxBaseAmt = taxProvider.calculateTaxBaseAmt(line.getAmount(), line.getTaxId());
```

Comment:
pls calculate taxBaseAmt as `line.getAmount().subtract(taxLine.getAmount())`.
(just to avoid rounding issues)